### PR TITLE
Add events page

### PR DESCRIPTION
**Pull request by matthewbrown77**
* Creates basic event page.
* Adds event page route to config.ts.

### CODE REVIEW

File: config.ts [hunk 1]
```diff
@@ -21,7 +21,8 @@ export default {
   pages: {
     index: "/",
     login: "/login",
-    donate: (path = "[id]"): string => `/donate/${path}`
+    donate: (path = "[id]"): string => `/donate/${path}`,
+    events: "/events/events"
```
Comment (matthewbrown77) [inline config.ts:25]: Is having an events subfolder here necessary? From my understanding, all of the other `events` front-end code will be under `components`, so there shouldn't be multiple event-related files under the `pages` directory.

File: pages/events/events.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { NextPage, GetServerSideProps } from "next";
+
+const EventsPage: NextPage = () => null;
```
Comment (matthewbrown77) [inline pages/events/events.tsx:4]: This is where the [root component](https://github.com/gtbitsofgood/gen-soln/issues/93) should go.

**Comment by matthewbrown77** [comment]
Forgot to attach page screenshot.
events_page

File: config.ts (hunk 1 above)
Comment (PC98) [reply config.ts:25]: So the idea is we will at-least have the following routes:
`/events/` -> to view all events
`/events/[id]` -> to view information about an event with a specific ID.

To support the above routes in Next.js, we will have to create a `pages/events/` folder.

**Review by PC98 — APPROVED** [review]
@matthewbrown77 Please rename your file to `pages/events/index.tsx` and make appropriate changes in `config.ts` -- we want to support the `events/` route and not `events/events/` 🙂

Feel free to merge after making this change.

**Comment by matthewbrown77** [comment]
> @matthewbrown77 Please rename your file to `pages/events/index.tsx` and make appropriate changes in `config.ts` -- we want to support the `events/` route and not `events/events/` 
> 
> Feel free to merge after making this change.

@PC98 Addressed in 1f4d668. Merging from previous approval.

**Comment by PC98** [comment]
I will follow up with @Arthelon about this -- even I am not a Vercel team member

**Pull request merged**